### PR TITLE
fix old library reference and related code

### PR DESCRIPTION
**Pull request by JohnDoe389**
minor fix to cashier check program

### CODE REVIEW

File: examples/cashiers-check/tests/cashiers-check.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 const anchor = require("@project-serum/anchor");
 const serumCmn = require("@project-serum/common");
-const TokenInstructions = require("@project-serum/serum").TokenInstructions;
 const assert = require("assert");
+const { TOKEN_PROGRAM_ID, Token } = require("@solana/spl-token");
```
Comment (armaniferrante) [inline examples/cashiers-check/tests/cashiers-check.js:4]: `Token` can be removed right?

Comment (JohnDoe389) [reply examples/cashiers-check/tests/cashiers-check.js]: true.   i just pushed a change to take it out.

**Review by armaniferrante — APPROVED** [review]

File: examples/lockup/tests/lockup.js [hunk 2]
```diff
@@ -116,6 +117,7 @@ describe("Lockup and Registry", () => {
         await lockup.state.rpc.whitelistAdd(e, { accounts });
       },
       (err) => {
+        console.log('errwhite', err);
```
Comment (armaniferrante) [inline examples/lockup/tests/lockup.js:120]:
```suggestion
```

File: examples/lockup/tests/lockup.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const assert = require("assert");
 const anchor = require("@project-serum/anchor");
 const serumCmn = require("@project-serum/common");
-const TokenInstructions = require("@project-serum/serum").TokenInstructions;
+const { TOKEN_PROGRAM_ID, Token } = require("@solana/spl-token");
```
Comment (armaniferrante) [inline examples/lockup/tests/lockup.js:4]:
```suggestion
const { TOKEN_PROGRAM_ID } = require("@solana/spl-token");
```

**Pull request merged**